bikeshed/boilerplate.py

```diff
@@ -109,7 +109,7 @@ def addSpecVersion(doc):
                     .decode(encoding="utf-8")
                     .strip()
                 )
-        except:
+        except FileNotFoundError:
```

Comment:
Pretty sure this should be a `subprocess.CalledProcessError` too.

Reply:
Swapped